changelogs/unreleased/5360-davinci26-minor.md

```diff
@@ -0,0 +1,5 @@
+## Adds critical level for access logging
+
+Similar motivation with https://github.com/projectcontour/contour/pull/4331/files
```

Comment:
The link to the files of old PR (which introduced error level) could be removed. The change log entry could be self-explanatory just by saying: 
> New `critical` access log level was introduced to reduce the volume of logs for busy installations. Critical level produces access logs for response status >= 500.

Reply:
done in fae4e657309748f248da06aecfdb217011e53299